tests/Integration/PrestaShopBundle/Controller/Admin/Order/DeliveryControllerTest.php

```diff
@@ -0,0 +1,134 @@
+<?php
```

Comment:
I like it a lot :)

Is it something you can document in prestafony-project? We could make the creation of tests required.

Also, for the already migrated pages we could add theses functional tests :)